### DISCUSSION TITLE
Interpreting "Toxicity" Plots

### DISCUSSION THREAD

**Discussion by isaacvock** [erhard-lab/grandR#21] [General]
Hello,

I like the idea behind grandR's `plotToxicityTest` family of functions, but I have concerns about the suggested interpretation encoded in the function name. I believe that there are explanations other than 4sU toxicity for correlations between NTR and log2-fold(+4sU/-4sU reads) and nothing in the preprint describing grandR rules out alternative explanations. I also believe it is in the metabolic labeling field's best interest to avoid stoking unnecessary fears of 4sU toxicity.

In addition, it is a bit unclear to me as to how 4sU cytotoxicity would lead to this correlation. If cells die at an elevated rate when fed 4sU, I would expect that to globally reduce levels of all RNAs in 4sU fed samples, regardless of their label status. Thus, without unlabeled spike-in RNA normalization, there would be no apparent difference in +4sU and -4sU RPMs.

Because of this, I would argue for changing the names of these functions to better reflect the information they provide (e.g., plot4sUDropout).

Thank you for considering my proposal,
Isaac

**Comment by florianerhard** [comment]
Dear Isaac,
thank you very much for your insight, this is very much appreciated! You are absolutely right that there are other reasons for these observations than effects of 4sU on living cells (in fact, we have a lot of data on this and an almost finished manuscript).

If you agree, we would actually like to directly go with your proposal and use `Plot4sUDropout` instead of `PlotToxicityTest`. I frankly never liked this name...

These plots analyze the correlation of the drop-out with the RNA half-life. A possible explanation for this is that transcription goes down globally, e.g. because Pol2 processivity is inhibited. In this scenario, the first genes that appear to be downregulated are the ones with short RNA-half-lives.

Best,
Florian

**Reply by isaacvock** [reply]
Hello Florian,

Yes, I fully support you all going with `Plot4sUDropout`, thank you for your receptiveness!

I see, your proposed mechanism makes sense. Thank you for the explanation, and I look forward to reading your manuscript.

Best,
Isaac
